tools/depends/configure.ac

```diff
@@ -266,6 +270,13 @@ case $host in
         if test "x$use_cpu" = "xauto"; then
           use_cpu=$host_cpu
```

Comment:
Is this `$host_cpu` really correct, make no sense on i*86 (32bit) to use my system 64bit :sweat_smile:

Reply:
What's the value of `host_cpu`?